packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -553,6 +553,21 @@ export const voicecraft = (model: ModelData): string[] => [
 model = VoiceCraft.from_pretrained("${model.id}")`,
 ];
 
+export const chattts = (model: ModelData): string[] => [
+	`import ChatTTS
+from IPython.display import Audio
```

Comment:
```suggestion
from IPython.display import Audio
import torchaudio
```

Make sure to add `torchaudio` otherwise the snippet would fail.

Reply:
+ remove `Audio` as it's not being used?